### PR TITLE
Update readme to add development information

### PR DESCRIPTION
**Pull request by Azurelol**
## Description

Update readme to add some notes on development workflow.

## Changes
* ![UPDATE] README.MD : Add some notes on development workflow.

[NEW]:https://resources.atlas.wooga.com/icons/icon_new.svg "New"
[ADD]:https://resources.atlas.wooga.com/icons/icon_add.svg "Add"
[IMPROVE]:https://resources.atlas.wooga.com/icons/icon_improve.svg "IMPROVE"
[CHANGE]:https://resources.atlas.wooga.com/icons/icon_change.svg "Change"
[FIX]:https://resources.atlas.wooga.com/icons/icon_fix.svg "Fix"
[UPDATE]:https://resources.atlas.wooga.com/icons/icon_update.svg "Update"

[BREAK]:https://resources.atlas.wooga.com/icons/icon_break.svg "Remove"
[REMOVE]:https://resources.atlas.wooga.com/icons/icon_remove.svg "Remove"
[IOS]:https://resources.atlas.wooga.com/icons/icon_iOS.svg "iOS"
[ANDROID]:https://resources.atlas.wooga.com/icons/icon_android.svg "Android"
[WEBGL]:https://resources.atlas.wooga.com/icons/icon_webGL.svg "WebGL"
[UNITY]:https://resources.atlas.wooga.com/icons/icon_unity.svg "Unity"
[GRADLE]:https://resources.atlas.wooga.com/icons/icon_gradle.svg "Gradle"
[LINUX]:https://resources.atlas.wooga.com/icons/icon_linux.svg "Linux"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,8 +127,16 @@ def testEnvironment = [ 'osx':
 buildGradlePlugin plaforms: ['osx','windows','linux'], testEnvironment: testEnvironment
 ```
 
+# Development
+
+Whenever work is done to update these pipelines, the changes are propagated like so:
+
+1. The changes are added as a feature PR based on `master`. They must be merged into `master`.
+2. A release must be made on the Jenkins CI for the job corresponding to this repository. As consequence, a new branch is made named after the newer version. 
+3. Any jobs that use the pipeline must do a run before the newer changes are loaded. The simplest is to trigger a snapshot build. Any builds after that should reflect the changes that the newer pipeline introduces.
```
Comment (Larusso) [inline README.md:136]: The wording suggests that any pipeline from this repo has a snapshot release stage whatever. I would rephrase to trigger a build with default parameters.

**Review by Larusso — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Azurelol) [reply README.md:136]: Good point.

Comment (Azurelol) [reply README.md]: Done.

**Review by Larusso — APPROVED** [review]
